pyproject.toml

```diff
@@ -67,6 +67,11 @@ test = [
     "typing-extensions>=4.3,<5",
     "tzdata; sys_platform == 'win32'",
 ]
+dev = [
+    "black==22.6",
+    "typing_extensions>=4.3,<5",
+    "pyright==1.1.351",
```

Comment:
Might be a questionable addition since pyright itself is actually an [npm package](https://www.npmjs.com/package/pyright). The python module uses [`nodeenv`](https://pypi.org/project/nodeenv/) which by the authors own admission is less stable than using `pyright[nodejs]` which uses [`nodejs-wheel`](https://pypi.org/project/nodejs-wheel-binaries/) in it's stead.

My suggestion would be omitting this entirely and letting the users use Pylance or their own installation of pyright.